### DISCUSSION TITLE
Pre-launch Feedback

### DISCUSSION THREAD

**Discussion by dcramer** [dcramer/peated#69] [Announcements]
I started working on Peated mostly out of the desire to build something, but I also wanted a simple way to keep what might as well be a "log" of whiskey/whisky/bourbons I had tried. I had used apps like Untappd and Vivino  in the past for similar things, but there didn't seem to be anything in the broader whiskey space that was well built. So here we are.

In its current state Peated lets you capture tasting notes (this is the log) and track favorites. It creates a similar social experience to Untappd (which I wildly prefer over Vivino), though I've tried to improve upon things where it made sense. There's a few other key areas that exist that still need more work:

- The database is key, so I've tried to make it somewhat easy to add bottles/distillers/etc. Still work to be done here to avoid duplicate entries, merge existing duplicates, etc. A lot of the data in the DB exists from either manual entry or a scraper I built.
- I built a basic price scraper that uses a few US sources, but there are areas it needs improved on matching. My main goal around this was to provide value tracking on bottles, as its always interesting to know if that random bottle you picked up several years ago has 10x'd in price.
- There's a variety of stability concerns yet, and some of the UI is not as responsive as I'd like. This area is a hard blocker to a real "launch".

Anyways, so I've got a list of what I feel are underwhelming featuers unless you're hardcore, or it achieves critical mass. I've been asking myself "what would make this great", so I'm here asking for input. I'm pretty capable of building whatever it is, but I feel like I'm missing that wow-factor, and part of that might simply be because I myself am an amateur at this hobby.

So if you've got feedback, something you think would be killer, or just general sentiment around what exists so far, I'd love to hear from you!

https://peated.app

**Comment by chuckcarpenter** [comment]
I'd like more info to add to the bottle, factual bits like proof and batch (given some barrel picks can differentiate from one another) and mash bill.

**Reply by dcramer** [reply]
Used to actually have this but it made things _sooo_ complicated. Tried two iterations:

1. Each bottle had this info, and each bottle was unique across this info (e.g. you would have two bottles in search for two unique bottles, barrelings)
2. Each bottle could _also_ add vintage information, which was stored with either collection info or tasting info

2 felt better but it still over complicated the experience

Open to ideas here though!

**Reply by dcramer** [reply]
Had one quick thought:

What if we just made an 'advanced mode'?

There's still the challenge of how do you select/find a bottle to add a tasting to (e.g. do all bottles of the same basic style go together, or are they split with barrel no etc), but we could have an advanced mode that enables:

- Vintage details
- Mash bill
- Segmented tasting notes (e.g. node/body/finish)
